README.md

```diff
@@ -363,7 +362,7 @@ def slaveTemplateUsEast1Parameters = [
   nodeProperties:                null
 ]
 
-def AmazonEC2CloudParameters = [
+def EC2CloudParameters = [
```

Comment:
Tested this Groovy scripts after this PR with both the old and the new class names. Confirmed XStream serialization was the same as before.